### PR TITLE
node: Add missing spaces

### PR DESCRIPTION
**Pull request by J-295**
Add missing spaces in lists of single-line code blocks.

### CODE REVIEW

**Comment by J-295** [comment]
I was originally planning to only do the changes in my first commit, but I am now finished.

File: types/node/v16/crypto.d.ts [hunk 1]
```diff
@@ -3392,7 +3392,7 @@ declare module "crypto" {
      */
     function getCipherInfo(nameOrNid: string | number, options?: CipherInfoOptions): CipherInfo | undefined;
     /**
-     * HKDF is a simple key derivation function defined in RFC 5869\. The given `ikm`,`salt` and `info` are used with the `digest` to derive a key of `keylen` bytes.
+     * HKDF is a simple key derivation function defined in RFC 5869\. The given `ikm`, `salt` and `info` are used with the `digest` to derive a key of `keylen` bytes.
      *
      * The supplied `callback` function is called with two arguments: `err` and`derivedKey`. If an errors occurs while deriving the key, `err` will be set;
```
Comment (codershiba) [inline types/node/v16/crypto.d.ts]: See line 3397, space missing before `derivedKey`

**Comment by J-295** [comment]
The bot says there is a merge conflict yet to me it says there are none.
image

**Comment by codershiba** [comment]
Remove the changes for v20 I have included those and even more of these spacing problems in #69719

**Review by peterblazejewicz — APPROVED** [review]

**Comment by J-295** [comment]
Ready to merge

**Review by peterblazejewicz — APPROVED** [review]

**Comment by J-295** [comment]
Ready to merge

**Pull request merged**